src/pe/mod.rs

```diff
@@ -92,6 +104,45 @@ impl<'a> PE<'a> {
         let mut certificates = Default::default();
         let mut is_64 = false;
         if let Some(optional_header) = header.optional_header {
+            #[cfg(feature = "pe_authenticode")]
+            let mut authenticode_sections_ = {
```

Comment:
I would add a comment `// Sections we are assembling through the parsing, eventually, it will be passed to the actual attribute of the structure.` or something along those lines.